### DISCUSSION TITLE
Installing oracle-jdk version 17

### DISCUSSION THREAD

**Discussion by ashr123** [Homebrew/discussions#2142] [Tap maintenance and Homebrew development] [answered]
hey, can you update the `oracle-jdk` installation to version 17?

**Comment by alebcay** [comment, marked as answer]
We're currently in the progress of testing `openjdk` 17 and ensuring that it does not break any of the packages in the Homebrew repo that depend on it. See https://github.com/Homebrew/homebrew-core/pull/85174 for progress/blockers.

**Reply by ashr123** [reply]
So if I understand correctly, when you ensure that `openjdk` is safe for use, it will also tell you that `oracle-jdk` 17 is safe for use and you'll update it accordingly?

**Reply by cho-m** [reply]
[`oracle-jdk`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/oracle-jdk.rb) is a Cask, so anyone can open a PR to update to v17 in https://github.com/Homebrew/homebrew-cask. There is no specific person who is assigned to update Casks/Formulae, so the update will depend on whenever a contributor (external or maintainer) has time to open a PR.

This is the same situation for all other JDK distribution Casks, like Zulu, Temurin, and Microsoft.

The update for `openjdk` Formula is separate from this and does not block you or any other user from opening a PR to update `oracle-jdk` Cask.

**Reply by ashr123** [reply]
I've looked there, and there isn't an existing issue template for updating an existing cask (only templates for Reproducible Bug Report or Feature Request), what should I do? how can I help the process of updating the `oracle-jdk` to version 17?

**Reply by SMillerDev** [reply]
https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask

**Reply by ashr123** [reply]
I did it, but I'm getting errors that I dont understand them, you can see it on Homebrew/homebrew-cask#111389
